hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyGenerator.java

```diff
@@ -1060,8 +1060,10 @@ public void run() {
             }
             is.close();
           }
-        } catch (IOException | InterruptedException ex) {
+        } catch (IOException ex) {
           LOG.error("Exception while validating write: " + ex.getMessage());
```

Comment:
should we change line 1064 like:
LOG.error("Exception while validating write.", ex);

Reply:
@xiaoyuyao addressed in latest commit.